src/classes/RecipeRepository.js

```diff
@@ -1,39 +1,38 @@
-import Ingredient from './Ingredient';
 import Recipe from './Recipe';
 
 class RecipeRepository {
   constructor(recipeData, ingredientData) {
     this.allRecipes = this.addRecipes(recipeData, ingredientData);
     this.tags = [];
-  }
+  };
 
   addRecipes(recipeData, ingredientData) {
     return recipeData.map((data) => {
-      return new Recipe(data, ingredientData)
-    })
-  }
+      return new Recipe(data, ingredientData);
+    });
+  };
 
   getTags() {
```

Comment:
Think we need to add recipeData in as parameter there. @michaelputnam67 can you please add and re-push?

Reply:
Fixed your request. Thank you for pointing this out!